packages/core/src/App/Containers/PasswordSelectionModal/password-selection-modal.jsx

```diff
@@ -28,22 +28,22 @@ const PasswordSelectionModal = observer(
         const { ui } = useStore();
         const { is_mobile } = ui;
 
+        const loggedIn = !!cacheTrackEvents.parseCookies('client_information');
```

Comment:
Remove this from here also and add it in globally

Reply:
Fixed.